### PR TITLE
feat: meta attributes for redesigned gallery item

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] add meta attributes for redesigned gallery item

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/gallery/GalleryItemHead.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div></div>
```
Comment (Jarsen136) [inline components/gallery/GalleryItemHead.vue:2]: How about writing a `useGalleryItemHead` hook or utils function for it, as there is no UI content in this component

**Review by Jarsen136 — COMMENTED** [review]
How could we test the seo content for gallery item? A screenshot would be better.

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
pay 50 usd 
merge when good to go

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 27.96 USD/KSM ~ 1.788 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0xb76f7e66708bbfdbfea9d88aa7e5586be2fd622a47282fb9c9c6b8706fed88db](https://kusama.subscan.io/extrinsic/0xb76f7e66708bbfdbfea9d88aa7e5586be2fd622a47282fb9c9c6b8706fed88db)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by Jarsen136** [comment]
> > Hey @preschian , could you also check why the cf page building fail?
> 
> this one because of my PR yesterday. let me sync with main first, should be fixed in this PR #4680

Yes, it works.

**Comment by Jarsen136** [comment]
Need one more approval to merge this PR. @roiLeo

**Review by roiLeo — APPROVED** [review]
🚀

**Pull request merged**